scripts/e2e/install-e2e-tests-deps.sh

```diff
@@ -24,5 +24,5 @@ gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub
 EOF
     sudo yum install -y google-chrome-stable
     sudo rm /etc/yum.repos.d/google-chrome.repo
-    sudo yum install -y python-requests pyOpenSSL python-jinja2 python-jwt
+    sudo yum install -y python-requests pyOpenSSL python-jinja2 python-jwt python-scipy
```

Comment:
```suggestion
    sudo yum install -y python-requests pyOpenSSL python-jinja2 python-jwt scipy
```

see https://centos.pkgs.org/7/centos-x86_64/scipy-0.12.1-6.el7.x86_64.rpm.html , scipy does not provide python-scipy. so we have to specify "scipy".

Reply:
Done